feature-group-definitions/simd.yml

```
@@ -0,0 +1,6 @@
+name: SIMD
+description: The 128-bit SIMD (Single Instruction Multiple Data) extension to WebAssembly performs one instruction on multiple units of data, when running on hardware that supports such instructions. Also known as vector instructions.
+spec: https://webassembly.github.io/spec/core/bikeshed/#vector-instructions%E2%91%A0
```

When there's this trailing stuff representing a circled number like ①, that's because there were multiple generated IDs that conflicted, that needed to be disambiguated. And happily, https://webassembly.github.io/spec/core/bikeshed/#vector-instructions is an anchor on the very same section:

```suggestion
spec: https://webassembly.github.io/spec/core/bikeshed/#vector-instructions
```